media/videostitcher/create_cdn_key.go

```diff
@@ -0,0 +1,91 @@
+// Copyright 2023 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package videostitcher
+
+// [START videostitcher_create_cdn_key]
+import (
+	"context"
+	"fmt"
+	"io"
+
+	stitcher "cloud.google.com/go/video/stitcher/apiv1"
+	stitcherstreampb "cloud.google.com/go/video/stitcher/apiv1/stitcherpb"
+)
+
+// createCDNKey creates a CDN key. A CDN key is used to retrieve protected media.
+// If isMediaCDN is true, create a Media CDN key. If false, create a Cloud
+// CDN key. To create a privateKey value for Media CDN, see
+// https://cloud.google.com/video-stitcher/docs/how-to/managing-cdn-keys#create-private-key-media-cdn.
+func createCDNKey(w io.Writer, projectID, keyID, hostname, keyName, privateKey string, isMediaCDN bool) error {
```

Comment:
Only pass in arguments if they are necessary for testing -- I think at least isMediaCDN could be a local var instead, and possibly hostname and keyID as well (since these are static in your tests). https://googlecloudplatform.github.io/samples-style-guide/#minimal-arguments

Reply:
Removed hostname and keyName from arguments. keyID and privateKey are needed since these IDs have a randomized string component for testing. isMedia is needed since the sample can create Media CDN keys or Cloud CDN keys during testing.